packages/web/src/scss/components/Toast/README.md

```diff
@@ -0,0 +1,247 @@
+
+1. On devices with rounded displays and/or notches (e.g. iPhone X and newer), the Toast component is pushed inwards to
+   avoid the rounded corners. The `viewport-fit="cover"` meta tag is required for this feature to work:
+
+   ```html
+   <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover" />
+   ```
+
+2. Android Chrome only: When the vertical alignment is set to `bottom` and the virtual keyboard is open, the Toast
+   component is pushed upwards to avoid being covered by the keyboard. This feature requires the following JavaScript
+   snippet and is currently supported only in Chrome 94 on Android and later.
+
+   ```js
+   // Enable CSS to detect the presence of virtual keyboard:
+   if ('virtualKeyboard' in navigator) {
+     navigator.virtualKeyboard.overlaysContent = true;
+   }
+   ```
+
+### Toast Queue
+
+When multiple ToastBar components are present, they stack up in a queue, separated by a gap. The ToastBar components are
+sorted from top to bottom for the `top` vertical alignment, and from bottom to top for the `bottom` vertical alignment.
+When the queue does not fit the screen, it becomes scrollable.
+
+👉 Please note that the initial scroll position is always at the **top** of the queue.
+
+👉 Please note that scrolling is only possible over the toast message boxes and not over the entire container, so the
+page behind the toast messages is still accessible.
+
+## ToastBar
+
+The ToastBar component is the actual toast notification. It is a simple container with a message and a few optional
+elements.
+
+Minimum example:
+
+```html
+<div class="ToastBar ToastBar--inverted">
+  <div class="ToastBar__content">
+    <div class="ToastBar__message">Message only</div>
+  </div>
+</div>
+```
+
+### Optional Icon
+
+An icon can be added to the ToastBar component:
+
+```html
+<div class="ToastBar ToastBar--inverted">
+  <div class="ToastBar__content">
+    <svg width="20" height="20" aria-hidden="true">
+      <use xlink:href="/icons/svg/sprite.svg#info" />
+    </svg>
+    <div class="ToastBar__message">Message with icon</div>
+  </div>
+</div>
+```
+
+### Action Link
+
+An action link can be added to the ToastBar component:
+
+```html
+<div class="ToastBar ToastBar--inverted">
+  <div class="ToastBar__content">
+    <div class="ToastBar__message">
+      Message with action
+      <a href="#" class="link-inverted link-underlined">Action</a>
+    </div>
+  </div>
+</div>
+```
+
+👉 **Do not put any important actions** like "Undo" in the ToastBar component (unless there are other means to perform
+said action), as it is very hard (if not impossible) to reach for users with assistive technologies. Read more about
+[Toast accessibility](#scott-o-hara-toast) at Scott O'Hara's blog.
+
+### Colors
+
+The ToastBar component is available in all [emotion colors][dictionary-color], plus the `inverted` variant (default).
```

Comment:
is it default if you have to specify it? :) or at least you do in the minimum example

Reply:
I know. For the `web` package it's all about "recommended defaults". All other technologies (including Figma) can have a default variant, but we always need to specify an option in the `web` package, even if it's communicated as "default".

I checked again other `web` components' docs and at least we have it the same everywhere.

Reply:
OK.